src/utils/miscellaneous.ts

```diff
@@ -1,4 +1,17 @@
 import { TokenDetails } from 'types'
+import { AssertionError } from 'assert'
+
+export function assert<T>(val: T, message: string): asserts val is NonNullable<T> {
```

Comment:
Consider dropping this function when DefinitelyTyped/DefinitelyTyped#40415 is fixed

Reply:
This actually asserts not only that val is NonNullable, but that it is truthy. No idea if it's possible to type that.